group1/comm/static/comm_tool.js

```diff
@@ -60,6 +61,34 @@ function startVideoChat() {
     alert("Please create a team to be able to video chat")
   }
 }
+*/
+
+
+//Set the sentinel for turning on/off the chat window
+let chatcounter = 0; 
+
+//Function for starting and stopping the video chat
+function startVideoChat(){
+  if(chatcounter == 0){
```

Comment:
Missing space between "if" and "(".  Missing space between ")" and "{"

Reply:
resolved